.github/workflows/release.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: "Publish Cibyl to PyPI"
+on:
+  push:
+    branches:
+      - main
+    tags:
+      # asterisk allow matching tags like v1.0.0.rc1, etc.
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
Passing the pattern through a parser tells me that it is not valid. I have played a little bit with it and ended up with: 'v[0-9]+\.[0-9]+\.[0-9]+.*'. This will match: 'v1.0.0.rc1'.

Tell me what you think about it.

Reply:
I'm not sure about this. With most regex flavours, you're probably right. But the examples in github docs suggest  that the dot is not a wildcard (e.g. https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet )

Reply:
Ok, that is some weird regex distro, but if it works like that then there is no problem.